Description: disable flaky test
 storecompacttest fails more often than not. also problem in upstream CI
Author: Harald Sitter <sitter@kde.org>
Origin: vendor
Bug: https://bugs.kde.org/show_bug.cgi?id=353365
Reviewed-by: Jonathan Riddell <jr@riddell.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: kdepim-runtime/resources/mixedmaildir/autotests/CMakeLists.txt
===================================================================
--- kdepim-runtime.orig/resources/mixedmaildir/autotests/CMakeLists.txt
+++ kdepim-runtime/resources/mixedmaildir/autotests/CMakeLists.txt
@@ -191,15 +191,15 @@ target_link_libraries(
 )
 
 # test for store compact handling
-add_executable(storecompacttest  ../mixedmaildirstore.cpp  ../mixedmaildirresource_debug.cpp
- storecompacttest.cpp
- ${testdata_SRCS}
-)
-add_test(storecompacttest storecompacttest)
-ecm_mark_as_test(mixedmaildir-storecompacttest)
-
-
-target_link_libraries(
-  storecompacttest
-  ${common_link_libraries}
-)
+# add_executable(storecompacttest  ../mixedmaildirstore.cpp  ../mixedmaildirresource_debug.cpp
+#  storecompacttest.cpp
+#  ${testdata_SRCS}
+# )
+# add_test(storecompacttest storecompacttest)
+# ecm_mark_as_test(mixedmaildir-storecompacttest)
+# 
+# 
+# target_link_libraries(
+#   storecompacttest
+#   ${common_link_libraries}
+# )
